wordpress-mobile/WordPress-iOS
# Update Aztec to version 1.15.0 and matching Gutenberg version.

Fixes #

To test:
 - Check the Aztec and Gutenberg editor to see if all is good
 - Test the header and blockquote functionality on both to see if all is working correctly

PR submission checklist:
